scenario/BulkerScenario.ts

```diff
@@ -8,7 +8,7 @@ import { exp } from '../test/helpers';
 scenario(
   'Comet#bulker > (non-WETH base) all non-reward actions in one txn',
   {
-    filter: async (ctx) => await isBulkerSupported(ctx) && !matchesDeployment(ctx, [{deployment: 'weth'}, {network: 'mumbai'}, { network: 'linea-goerli' }]),
+    filter: async (ctx) => await isBulkerSupported(ctx) && !matchesDeployment(ctx, [{deployment: 'weth'}, {network: 'mumbai'}, { network: 'linea-goerli' }, { network: 'optimism' }]),
```

Comment:
So based on the comment on line 7, this is failing because `WETH` is defined as the first collateral asset in `configuration.json`. I think it would be good to keep this test coverage, so IMO it would be better to define another asset (e.g. `OP`) as the first collateral asset and remove this filter.